test-scripting/techs/Categories.inf.py

```diff
@@ -1,44 +1,17 @@
-Category(
-    name="LEARNING_CATEGORY",
-    graphic="learning.png",
-    colour=(54, 202, 229))
-
-Category(
-    name="GROWTH_CATEGORY",
-    graphic="growth.png",
-    colour=(116, 225, 107))
-
-Category(
-    name="PRODUCTION_CATEGORY",
-    graphic="production.png",
-    colour=(240, 106, 106, 255))
-
-Category(
-    name="CONSTRUCTION_CATEGORY",
-    graphic="construction.png",
-    colour=(241, 233, 87, 255))
-
-Category(
-    name="DEFENSE_CATEGORY",
-    graphic="defense.png",
-    colour=(70, 80, 215, 255))
-
-Category(
-    name="SHIP_HULLS_CATEGORY",
-    graphic="ship_hulls.png",
-    colour=(255, 139, 85, 255))
-
-Category(
-    name="SHIP_PARTS_CATEGORY",
-    graphic="ship_parts.png",
-    colour=(255, 139, 85, 255))
-
-Category(
-    name="SHIP_WEAPONS_CATEGORY",
-    graphic="ship_weapons.png",
-    colour=(255, 139, 85, 255))
-
-Category(
-    name="SPY_CATEGORY",
-    graphic="spy.png",
-    colour=(168, 0, 255, 255))
+Category(name="LEARNING_CATEGORY", graphic="learning.png", colour=(54, 202, 229))
```

Comment:
I think this was more readable before the reformat

Reply:
This line fits in one line (120) so it is inlined. 

This is the disadvantage of such tools.  They are better only for 90% of cases.

Reply:
I went to docs to answer other questions and found that this could be fixed.

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma

I will fix it tomorrow.